vmdb/config/vmdb.tmpl.yml

```diff
@@ -288,6 +288,7 @@ workers:
         :duration: 10.seconds
         :capacity: 50
         :amqp_port: 5672
+        :event_catcher_openstack_infra:
```

Comment:
This might have to be defined as `:event_catcher_openstack_infra: {}`  otherwise that key represents nil, which is not what you'd expect

```ruby
[4] pry(main)> YAML.load("---\n:event_catcher_openstack_infra:\n")
=> {:event_catcher_openstack_infra=>nil}
[5] pry(main)> YAML.load("---\n:event_catcher_openstack_infra: {}\n")
=> {:event_catcher_openstack_infra=>{}}
```